src/m365/spo/commands/homesite/homesite-remove.spec.ts

```diff
@@ -144,10 +215,59 @@ describe(commands.HOMESITE_REMOVE, () => {
     });
 
     await command.action(logger, { options: { force: true } });
-    assert(homeSiteRemoveCallIssued);
+    assert(postStub.calledOnce);
+    assert.deepStrictEqual(postStub.lastCall.args[0].url, "https://contoso-admin.sharepoint.com/_api/SPO.Tenant/RemoveTargetedSite");
+  });
+
+  it('removes the Home Site specified by URL', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://contoso-admin.sharepoint.com/_api/SPO.Tenant/GetTargetedSitesDetails`) {
+        return homeSites;
+      }
+
+      throw 'Invalid request';
+    });
```

Comment:
Here we stub the request to get site properties, while on the line below, we stub the util function to get the properties. Let's use the one below and get rid of this stub since it's not used anyway.